README.md

```diff
@@ -10,15 +10,18 @@ Painless peer-to-peer WebRTC networking for rust's native and wasm applications.
 The goal of the Matchbox project is to enable udp-like, unordered, unreliable
 p2p connections in web browsers or native to facilitate low-latency multiplayer games.
 
-- [Introductory blog post](https://johanhelsing.studio/posts/introducing-matchbox)
-- [Tutorial for usage with Bevy and GGRS](https://johanhelsing.studio/posts/extreme-bevy)
```

Comment:
I've went ahead and removed these links as the content is now far enough out of date it would be likely to cause more confusion than anything else. Would very much like to discuss this. At what point to we make a "book"?

Reply:
I think Johan said he wanted to update it

Reply:
Excellent! That's really the ideal solution

Reply:
Yeah the tutorial will be at the same url, so leave it in there.

It's almost ready, will push the updated version as i publish the new crates.

Reply:
The intro too?

Reply:
Maybe not. I'm not really sure how the readme stands on its own, maybe there are some parts that need to be fleshed out a little.

Reply:
Removed the intro